nmtwizard/framework.py

```diff
@@ -1356,17 +1359,20 @@ def bundle_dependencies(objects, config, local_config, keep_all=False):
         return config
     else:
         if isinstance(config, str):
+            filename = None
+            train_str = ""
             if os.path.isabs(config) and os.path.exists(config):
                 filename = os.path.basename(config)
             else:
                 match = utility.ENVVAR_ABS_RE.match(config)
-                if match and ("TRAIN" not in match.group(1) or keep_all):
-                    filename = match.group(2)
-                else:
-                    filename = None
+                if match:
+                    if "TRAIN" in match.group(1):
+                        train_str = "_TRAIN"
+                    if not train_str or keep_all:
+                        filename = match.group(2)
             if filename is not None:
                 objects[filename] = local_config
-                return "${MODEL_DIR}/%s" % filename
+                return "${MODEL%s_DIR}/%s" % (train_str, filename)
```

Comment:
It's not immediately clear to me why this is needed. The "TRAIN" marker is generally useful to not download training resources during inference, but here the files are already in the model.

Is this fixing an issue? If yes, can a test be added?

Reply:
As discussed, this is for release of standalone models. I added a test.